app/controlplane/internal/usercontext/attjwtmiddleware/attmiddleware_test.go

```diff
@@ -126,8 +128,9 @@ func TestAttestationAPITokenProvider(t *testing.T) {
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
 			ctx := transport.NewServerContext(context.Background(), &mockTransport{reqHeader: tc.tokenHeader})
+			logger := log.NewStdLogger(io.Discard)
```

Comment:
Is this a leftover?

Reply:
No, a logger is now required by the use case